app/helpers/field_slips_helper.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module FieldSlipsHelper
+  def last_observation
+    return unless User.current
+
+    ObservationView.last(User.current)
+  end
+end
```

Comment:
This is potentially a generally useful helper, but just remember that all helpers are in a global "helper namespace". If it's just for field slips, you may want a more specific name.

Reply:
I think this could be helpful in other places and it is not specific to FieldSlips.  I evolved it to create ObservationView.last(User) which might be a better standard to use than having it as a helper.  I don't have a strong opinion here.